target/linux/ramips/dts/mt7621_dual-q_h721.dts

```diff
@@ -21,23 +21,6 @@
 		bootargs = "console=ttyS0,115200";
 	};
 
-	gpio-export {
-		compatible = "gpio-export";
-		#size-cells = <0>;
-
-		usb-30-power {
-			gpio-export,name = "usb-30-power";
-			gpio-export,output = <0>;
```

Comment:
`gpio-export,output = <0>;` should be converted to `GPIO_ACTIVE_LOW` without `enable-active-high;` regulator.

Reply:
Done.